### PR TITLE
zigotica userspace add raw_hid_receive

### PR DESCRIPTION
**Pull request by zigotica**
## Description

zigotica userspace (updated): add raw_hid_receive to benefit from [active-app-qmk-layer-updater](https://github.com/zigotica/active-app-qmk-layer-updater) node script (run by the host system) to change layers programmatically depending on the current active app

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

none

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zigotica** [comment]
Hi @drashna thank you for approving the PR and requesting another review 4 weeks ago. Is there something I need to do to bring this to another Collaborator attention? I would love this to be merged :)

File: keyboards/z12/keymaps/zigotica/keymap.c
Comment (fauxpark) [inline keyboards/z12/keymaps/zigotica/keymap.c:85]: Optionally can be replaced with:
```suggestion
                SEND_STRING("\e:vnew\n");
```
etc.

Comment (zigotica) [reply keyboards/z12/keymaps/zigotica/keymap.c:85]: I like that, thank you!

**Comment by zigotica** [comment]
I merged the suggestion, anything else I need to address before the second approval? Thank you

**Review by tzarc — APPROVED** [review]

**Pull request merged**
